src/templates/collaboration/index.html.twig

```diff
@@ -0,0 +1,19 @@
+{% extends 'base.html.twig' %}
+
+{% block main %}
+<img src="{{ asset('images/Collaborations.jpg') }}"style="width:400x=;height:800px;"></a>
+<img src="{{ asset('images/ballin.png')   }}"style="width:190x=;height:150px;position:center;"></a>
```

Comment:
The styles should go to the stylesheet. Inline styles like these are discouraged.